vapi/simulator/simulator.go

```diff
@@ -829,7 +832,7 @@ func (s *handler) library(w http.ResponseWriter, r *http.Request) {
 			spec.Library.CreationTime = types.NewTime(time.Now())
 			spec.Library.LastModifiedTime = types.NewTime(time.Now())
 			dir := libraryPath(&spec.Library, "")
-			if err := os.Mkdir(dir, 0750); err != nil {
+			if err := os.Mkdir(dir, 0o750); err != nil {
```

Comment:
Is this an intended change? (same below)

Reply:
Yes, let's not touch this in this PR

Reply:
reverted the changes to
```
if err := os.Mkdir(dir, 0750); err != nil {
```